src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -19,36 +25,102 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
-### Verify artifact names match sourceMappingURL
+### Verify artifact names match `sourceMappingURL` value
 
-When [uploading source maps to Sentry](#uploading-source-maps-to-sentry), you must name your source map files with the same name found in `sourceMappingURL`.
+The `sourceMappingURL` comment on the last line of your bundled or minified JavaScript file tells Sentry (or the browser) where to locate the corresponding source map. This can either be a fully qualified URL, a relative path, or the filename itself. When uploading artifacts to Sentry, you must name your source map files with the same name found in the  `sourceMappingURL` comment.
```

Comment:
> When uploading artifacts to Sentry, you must name your source map files with the same name found in the  `sourceMappingURL` comment.

This is misleading, because it's not the same valued ... it's what the value resolves to, which is what the examples below try to explain. Can we clarify this?